js/src/wp-seo-post-scraper.js

```diff
@@ -246,19 +246,32 @@ var tmceHelper = require( './wp-seo-tinymce' );
 	 */
 	PostScraper.prototype.saveScores = function( score ) {
 		var indicator = getIndicatorForScore( score );
+		var $metaboxTrafficLight = $( '.wpseo-metabox-sidebar .yst-traffic-light' );
+		var $metaboxTrafficLightLink = $metaboxTrafficLight.closest( '.wpseo-meta-section-link' );
 
 		if ( tabManager.isMainKeyword( currentKeyword ) ) {
 			document.getElementById( 'yoast_wpseo_linkdex' ).value = score;
 
 			if ( '' === currentKeyword ) {
 				indicator.className = 'na';
 				indicator.screenReaderText = app.i18n.dgettext( 'js-text-analysis', 'Enter a focus keyword to calculate the SEO score' );
+				indicator.fullText = app.i18n.dgettext( 'js-text-analysis', 'Content optimization: Enter a focus keyword to calculate the SEO score' );
```

Comment:
Hm not sure, at the moment it is used for the title attribute and I guess it will be used for a "tooltip" in the future, where a final period would maybe look a bit weird?
<img width="342" alt="screen shot 2016-06-14 at 16 31 56" src="https://cloud.githubusercontent.com/assets/1682452/16046782/ea2efae8-324d-11e6-91c1-09444a1894c0.png">